Source/BSN.Commons/Infrastructure/MessageBroker/EventContracts/EventAggregator/Contracts/EventBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BSN.Commons.Infrastructure.MessageBroker.EventContracts.EventAggregator.EventModels;
+
+namespace BSN.Commons.Infrastructure.MessageBroker.EventContracts.EventAggregator.Contracts
+{
+    public abstract class EventBase<T> : IEvent<T> where T : IEventDataModel
```

Comment:
Please add comments for all classes and interfaces and reference to architecture document

Reply:
done.

Reply:
Please refer to Source/BSN.Commons/Infrastructure/MessageBroker/README.md instead of external links